### PR TITLE
updated the genus not found label

### PR DESCRIPTION
**Pull request by alanjleonard**

### CODE REVIEW

**Comment by alanjleonard** [comment]
[better support adding a new species when genus does not exist](https://trello.com/c/4TfkMpdt/539-better-support-adding-a-new-species-when-genus-does-not-exist)

File: src/language.js [hunk 1]
```diff
@@ -203,7 +203,17 @@ const loadingIndicator = {
 
 const createNewOptionModal = {
   addNewAttributeTitle: (attribute) => `Add New ${attribute}`,
+  genusNotFound: () => (
+    <>
+      Genus not found in MERMAID
+      <br />
+      <a target="_blank" href="https://datamermaid.org/contact-us">
```
Comment (mmnoo) [inline src/language.js:210]: Technically, the language file should have no logic other than some minor html for formatting, (bold italic, spans with class names). 

This is fine for now though. Someone can change it later.

Comment (alanjleonard) [reply src/language.js:210]: What's your recommendation for doing this the correct way?

File: src/App/integrationTests/App.dontshowOfflineDeleted.test.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ test('Collect page only shows records that arent marked to be deleted next sync'
 
   // show all the records
   await user.selectOptions(pageSizeSelector, '20')
-  await user.selectOptions(await screen.findByTestId('page-size-selector'), '20')
+  // await user.selectOptions(await screen.findByTestId('page-size-selector'), '20')
```
Comment (mmnoo) [inline src/App/integrationTests/App.dontshowOfflineDeleted.test.js:41]: Hmm, this looks like a typo. Ill check it out. pls dont merge yet

**Pull request merged**